### PR TITLE
chore: add changelog for v0.11.0

### PR DESCRIPTION
**Pull request by vanbasten17**
Added changelog for v0.11.0 for creating a release tag in github history (will be 0.11.0 tag).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,32 +1,112 @@
...
+  - Fixed noopener to avoid security risk in location links.
+  - Fixed `compromise` import in `@botonic/cli`'s `nlu` template.
+  - Webviews are now rendered within `div` instead of `iframe` for a better visualization when developing locally without affecting production builds
+
 ## [0.10.1] - 2020-14-01
+
 ### Added
-- Added default config for every template `{  defaultDelay: 0.4,  defaultTyping: 0.6  }` to avoid extravagant warnings in `botonic serve`.
+
+- Added default config for every template `{ defaultDelay: 0.4, defaultTyping: 0.6 }` to avoid extravagant warnings in `botonic serve`.
```
Comment (dpinol) [inline CHANGELOG.md:92]: "extravagant" sounds weird for a technical doc. maybe "confusing", "misleading"?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,32 +1,112 @@
...
+
+* [@botonic/core](https://www.npmjs.com/package/@botonic/core)
+
+  - New inspector to debug route matching.
+
+- [@botonic/react](https://www.npmjs.com/package/@botonic/react)
+
+  - Type definitions improvement in `index.d.ts`.
+  - New unit tests.
+  - New webchat features:
```
[the remark this answers is not in the document]
Comment (vanbasten17) [reply CHANGELOG.md:37]: it was like this, maybe I have broken the format by mistake, thanks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,32 +1,112 @@
...
+  - Error reporting decorator now also reports the originating exception.
+  - Flag to disable the CMS delivery cache.
+- [@botonic/plugin-dynamodb](https://www.npmjs.com/package/@botonic/plugin-dynamodb)
+  - Upgrade to typescript 3.8
+  - **Breaking change**: Move Env to its own file to ease tree-shaking
+
+### Fixed
+
+- [@botonic/react](https://www.npmjs.com/package/@botonic/react)
+  - Noto Sans JP now does not show up as default typography.
```
[the remark this answers is not in the document]
Comment (vanbasten17) [reply CHANGELOG.md:81]: yes, but not the expected one. At the beginnings we specified Noto Sans JP font as the main one

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,32 +1,112 @@
...
+  - All styles of the project are converted into [Styled Components](https://styled-components.com/).
+
+- [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
+  - Upgrade to typescript 3.8
+  - **Breaking change**: Refactor TopContent and Content base classes. ModelType renamed to TopContentType.
+  - **Breaking change**: Refactor so that SearchResult now returns ContentId's instead of ContentCallback's.
+  - Now only the required language-specific [NLP.js](https://www.npmjs.com/package/node-nlp) folders are imported. This enables tree shaking to exclude big sentiment analysis files.
+  - Error reporting decorator now also reports the originating exception.
+  - Flag to disable the CMS delivery cache.
+- [@botonic/plugin-dynamodb](https://www.npmjs.com/package/@botonic/plugin-dynamodb)
```
[the remark this answers is not in the document]
Comment (vanbasten17) [reply CHANGELOG.md:74]: better, fixed!

**Pull request merged**